### DISCUSSION TITLE
i have graphs but some outputs are invalid

### DISCUSSION THREAD

**Discussion by lt72884** [OpenFAST/openfast#1649] [Q&A]
Ok, i now have matlab generating graphs of the parameters from the outlist. So far i am getting some decent results. I have added some of my own parameters from the xls file to the elastodynBDoutputs.dat file and have sucess. However, i have noticed some default parameters provide a graph that says invlalid. Some of the input parameters i add to the out list, end up coming back as invlaid as well. It seems that the pattern is anything to do with force. Kn-m and others work just fine, but anything with newtons or Kn always yields invalid. 

Here is a the list i have added to the outlist section:

"LSSGagMya"
"Spn1FLxb1"
"Spn1FLyb1"
"Spn1FLzb1"
"Spn2FLxb1"
"Spn2FLyb1"
"Spn2FLzb1"
"Spn3FLxb1"
"Spn3FLyb1"
"Spn3FLzb1"
"Spn4FLxb1"
"Spn4FLyb1"
"Spn4FLzb1"
"Spn5FLxb1"
"Spn5FLyb1"
"Spn5FLzb1"
"Spn6FLxb1"
"Spn6FLyb1"
"Spn6FLzb1"			  
"BldPitch1"               - Blade 1 pitch angle
"Azimuth"                 - Blade 1 azimuth angle
"RotSpeed"                - Low-speed shaft and high-speed shaft speeds
"GenSpeed"                - Low-speed shaft and high-speed shaft speeds
"TTDspFA"                 - Tower fore-aft and side-to-side displacements and top twist
"TTDspSS"                 - Tower fore-aft and side-to-side displacements and top twist
"RootFzb1"
"RootMIP1"
"RootMzb1"
"TTDspTwst"               - Tower fore-aft and side-to-side displacements and top twist
"RotTorq"                 - Rotor torque and low-speed shaft 0- and 90-bending moments at the main bearing
"LSSGagMya"               - Rotor torque and low-speed shaft 0- and 90-bending moments at the main bearing
"LSSGagMza"               - Rotor torque and low-speed shaft 0- and 90-bending moments at the main bearing
"YawBrFxp"                - Fore-aft shear, side-to-side shear, and vertical forces at the top of the tower (not rotating with nacelle yaw)
"YawBrFyp"                - Fore-aft shear, side-to-side shear, and vertical forces at the top of the tower (not rotating with nacelle yaw)
"YawBrFzp"                - Fore-aft shear, side-to-side shear, and vertical forces at the top of the tower (not rotating with nacelle yaw)
"YawBrMxp"                - Side-to-side bending, fore-aft bending, and yaw moments at the top of the tower (not rotating with nacelle yaw)
"YawBrMyp"                - Side-to-side bending, fore-aft bending, and yaw moments at the top of the tower (not rotating with nacelle yaw)
"YawBrMzp"                - Side-to-side bending, fore-aft bending, and yaw moments at the top of the tower (not rotating with nacelle yaw)
"TwrBsFxt"                - Fore-aft shear, side-to-side shear, and vertical forces at the base of the tower (mudline)
"TwrBsFyt"                - Fore-aft shear, side-to-side shear, and vertical forces at the base of the tower (mudline)
"TwrBsFzt"                - Fore-aft shear, side-to-side shear, and vertical forces at the base of the tower (mudline)
"TwrBsMxt"                - Side-to-side bending, fore-aft bending, and yaw moments at the base of the tower (mudline)
"TwrBsMyt"                - Side-to-side bending, fore-aft bending, and yaw moments at the base of the tower (mudline)
"TwrBsMzt"           

Now, when i run the .fst file it works fine, no errors. BUT the .sum file shows that some of these parameters are invalid. 

0   Time                 (s)                 
   1   LSSGagMya            (kN-m)              
   2   Spn1FLxb1            INVALID             
   3   Spn1FLyb1            INVALID             
   4   Spn1FLzb1            INVALID             
   5   Spn2FLxb1            INVALID             
   6   Spn2FLyb1            INVALID             
   7   Spn2FLzb1            INVALID             
   8   Spn3FLxb1            INVALID             
   9   Spn3FLyb1            INVALID             
  10   Spn3FLzb1            INVALID             
  11   Spn4FLxb1            INVALID             
  12   Spn4FLyb1            INVALID             
  13   Spn4FLzb1            INVALID             
  14   Spn5FLxb1            INVALID             
  15   Spn5FLyb1            INVALID             
  16   Spn5FLzb1            INVALID             
  17   Spn6FLxb1            INVALID             
  18   Spn6FLyb1            INVALID             
  19   Spn6FLzb1            INVALID             
  20   BldPitch1            (deg)               
  21   Azimuth              (deg)               
  22   RotSpeed             (rpm)               
  23   GenSpeed             (rpm)               
  24   TTDspFA              (m)                 
  25   TTDspSS              (m)                 
  26   RootFzb1             INVALID             
  27   RootMIP1             INVALID             
  28   RootMzb1             INVALID             
  29   TTDspTwst            (deg)               
  30   RotTorq              (kN-m)              
  31   LSSGagMya            (kN-m)              
  32   LSSGagMza            (kN-m)              
  33   YawBrFxp             (kN)                
  34   YawBrFyp             (kN)                
  35   YawBrFzp             (kN)                
  36   YawBrMxp             (kN-m)              
  37   YawBrMyp             (kN-m)              
  38   YawBrMzp             (kN-m)              
  39   TwrBsFxt             (kN)                
  40   TwrBsFyt             (kN)                
  41   TwrBsFzt             (kN)                
  42   TwrBsMxt             (kN-m)              
  43   TwrBsMyt             (kN-m)              
  44   TwrBsMzt             (kN-m)    
  
  how do i make these NOT invalid? Why are they invalid?
  
  thanks.

**Comment by jjonkman** [comment]
Dear @lt72884,

Regarding outputs showing up as INVALID, you asked and I answered a similar question in the following discussion: https://github.com/OpenFAST/openfast/discussions/1627#discussioncomment-6231717.

Best regards,
